src/js/jsx/sections/transform/Size.jsx

```diff
@@ -42,7 +43,36 @@ define(function (require, exports, module) {
     var Size = React.createClass({
         mixins: [FluxMixin],
 
-        shouldComponentUpdate: function (nextProps) {
+        /**
+         * Variables for storing inital scrub value to determine correct offset.
+         * @private 
+         * @type {number}
+         */
+        _initialScrubW: null,
```

Comment:
Scrubby mixin will send the distance since the initial mousedown, so as we call different actions for each scrubby, model gets updated, and the state changes. This allows us to keep scrubbing continuous. I am not a fan of it either, but consdiering we throttle the action calls, and state changes, this is the most straightforward way. I'm open to suggestions.